src/Helper/ProcessHelper.php

```diff
@@ -28,9 +30,9 @@ class ProcessHelper extends Helper implements OutputAwareInterface
     protected $output;
 
     /**
-     * @var array
+     * @var SfProcessHelper
      */
-    protected $cache = [];
```

Comment:
This is also removed in #578 but I decided to do it here also so it's easier to rebase :)
Still need the other pr though, as it does more.